### PR TITLE
Fix things missed in last PR

### PR DESCRIPTION
**Pull request by scotthain**

### CODE REVIEW

**Comment by schisamo** [comment]
The only other things I would suggest are:

* Should we expose an attribute for the package version? Perhaps `toolchain_version`?
* If we think `#1` is a good idea does it make sense to shorten `toolchain_package` to `toolchain_name`?

**Comment by scotthain** [comment]
@schisamo added the version and renamed, I like these ideas. Makes it much easier to test new versions of the toolchain as well.

File: attributes/default.rb [hunk 1]
```diff
@@ -19,7 +19,8 @@
   omnibus['build_user']      = 'omnibus'
   omnibus['build_user_home'] = nil
   omnibus['ruby_version']    = '2.1.5'
-  omnibus['toolchain_package'] = 'omnibus-toolchain'
+  omnibus['toolchain_name'] = 'omnibus-toolchain'
+  omnibus['toolchain_version'] = '0.0.1'
```
Comment (schisamo) [inline attributes/default.rb]: Small style thing. Please realign the `=`'s.

File: spec/recipes/git_spec.rb [hunk 2]
```diff
@@ -73,6 +73,7 @@
       # data for the latter.
       ChefSpec::ServerRunner.new(platform: 'solaris2', version: '5.11') do |node|
         node.automatic['platform_version'] = '5.10'
+        node.default['omnibus']['toolchain_name'] = 'omnibus-toolchain'
```
Comment (schisamo) [inline spec/recipes/git_spec.rb]: Is this needed? Isn't this default set in the attributes file?

**Pull request merged**